site/homepage/layouts/_default/single.html

```diff
@@ -1,55 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Deletion of this file has some impact on the child pages of `Community`. All the pages under `Community` have an invalid date stamp as highlighted in the below image. Restoring the file `single.html` resolves it.

![image](https://user-images.githubusercontent.com/18659982/108186209-65b7f400-710d-11eb-875d-67b3456c6628.png)

Reply:
IMHO this should be fixed by setting the correct content type in the front-matter.